### PR TITLE
setting up package.json and grunt and sass set up

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: theme/sass/components/_testimonials.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* Testimonials */
+
+.testimonial-container.swiper-container {
+    & .full-width {
```
Comment (ElXyfer) [inline theme/sass/components/_testimonials.scss]: there shouldn't be a space between ampersand and .full-width

File: theme/sass/components/_text-and-image-component.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                filter: grayscale(100%);
+                transition: all .4s ease-in-out;
+            }
+        }
+    }
+}
+
+
+.text-and-image-component {
+    &.left {
```
Comment (ElXyfer) [inline theme/sass/components/_text-and-image-component.scss]: Everything inside and including &.left { } can go up to line 6. and also do the same for &.right {} (put it after left).

File: theme/sass/components/_testimonials.scss (hunk 1 above)
Comment (katlazarova) [reply theme/sass/components/_testimonials.scss]: done

File: theme/sass/components/_text-and-image-component.scss (hunk 2 above)
Comment (katlazarova) [reply theme/sass/components/_text-and-image-component.scss]: done

File: theme/sass/components/_header.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        li {
+            display: flex;
+            flex-grow: 1;
+            align-items: center;
+            justify-content: center;
+            height: 50px;
+        }
+
+        >a {
```
Comment (ElXyfer) [inline theme/sass/components/_header.scss]: Add a space after >

File: theme/sass/components/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        .footer-info-links {
+            display: flex;
+            flex: 0 0 25%;
+            max-width: 25%;
+            justify-content: center;
+            font-size: 14px;
+            padding: 0;
+            color: white;
+
+            &:hover {
```
Comment (ElXyfer) [inline theme/sass/components/_footer.scss]: Hover should be insider anchor tag not their container.

File: theme/sass/components/_header.scss (hunk 3 above)
Comment (katlazarova) [reply theme/sass/components/_header.scss]: done

File: theme/sass/components/_footer.scss (hunk 4 above)
Comment (katlazarova) [reply theme/sass/components/_footer.scss]: done

**Pull request merged**